.github/workflows/job-ocm.yml

```diff
@@ -0,0 +1,68 @@
+name: OCM
+on: [push, workflow_dispatch, pull_request]
```

Comment:
If I understand OCM correctly I guess we should not push an image on `pull_request`?

Reply:
I only left the `workflow_call` trigger in for now. If needed later, we can add the `workflow_dispatch` trigger back in.